mod/worker.php

```diff
@@ -41,8 +44,8 @@ function worker_init()
 		// But since it doesn't destroy anything, we just try to get more execution time in any way.
 		set_time_limit(0);
 
-		$fields = ['executed' => DateTimeFormat::utcNow(), 'pid' => getmypid(), 'done' => false];
-		$condition =  ['id' => $r[0]["id"], 'pid' => 0];
+		$fields = ['executed' => DateTimeFormat::utcNow(), 'process' => Process::getID(), 'done' => false];
```

Comment:
Ah, you have `getID()` and `getCode()` ... Hmm ...

Reply:
`getID()` is for the unique ID and `getCode()` is for the concatenated string :-)

I tried to encapsulate the Process Core and just add some necessary methods